src/main/java/net/minecraftforge/common/ForgeMod.java

```diff
@@ -223,4 +215,10 @@ public void registerRecipeSerialziers(RegistryEvent.Register<IRecipeSerializer<?
         event.getRegistry().register(new ConditionalRecipe.Serializer<IRecipe<?>>().setRegistryName(new ResourceLocation("forge", "conditional")));
 
     }
+
+    @OnlyIn(Dist.CLIENT)
```

Comment:
Do not use `@OnlyIn`.

Reply:
I used it it because the server was crashing. Ok I will just remove this.

Reply:
https://mcforge.readthedocs.io/en/latest/concepts/sides/